File: 1007/bee1007.java
// 1007 - Diferença
import java.io.IOException;
import java.util.Scanner;

class bee1007 {

    public static void main(String[] args) throws IOException {
        Scanner scan = new Scanner(System.in);

        int a, b, c, d, diferenca;
            a = scan.nextInt();
            b = scan.nextInt();
            c = scan.nextInt();
            d = scan.nextInt();

        diferenca = (a*b - c*d);

        System.out.println("DIFERENCA = " + diferenca);

        scan.close();
    }
}

File: 1007/bee1007.c
// 1007 - Diferença
#include <stdio.h>

int main(void) {

    int a, b, c, d, diferenca;
    scanf("%d%d%d%d", &a,&b, &c, &d);

    diferenca = (a*b - c*d);

    printf("DIFERENCA = %d\n", diferenca);

    return 0;
}